docs/standard/class-library-overview.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: ".NET Framework Class Library Overview"
+title: ".NET Class Library Overview"
 ms.custom: ""
 ms.date: "03/30/2017"
```

Comment:
update ms.date?

Reply:
@mairaw, of course. I always forget about updating the date.